compiler/ilgen/JitBuilderRecorderBinaryBuffer.hpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef TR_JITBUILDERRECORDER_BINARYBUFFER_INCL
+#define TR_JITBUILDERRECORDER_BINARYBUFFER_INCL
+
+#include "ilgen/OMRJitBuilderRecorderBinaryBuffer.hpp"
+
+namespace TR
+{
+   class JitBuilderRecorderBinaryBuffer : public OMR::JitBuilderRecorderBinaryBuffer
+      {
+      public:
+      //    JitBuilderRecorderBinaryBuffer(const TR::MethodBuilderRecorder *mb)
```

Comment:
Rather than commenting out the intended call because there is no longer a `TR::MethodBuilderRecorder` class, please just change it to plain old `TR::MethodBuilder` because I think that's what it should be anyway.